### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by M4RKL4B0Y**
Updated config file, added a lists_controller.js and updated tasks_controller.js

### CODE REVIEW

File: server/controllers/lists_controller.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// DEPENDENCIES
+const lists = require('express').Router()
+const db = require('../models')
+const { Lists } = db
```
Comment (tan-man21) [inline server/controllers/lists_controller.js:4]: Lists will need to be changed to "List
 singular like tasks.

**Review by tan-man21 — CHANGES_REQUESTED** [review]
In lists controller Lists needs to be changed to be singular List

**Review by M4RKL4B0Y — COMMENTED** [review]
Good eye, Thanks Tan-Man

**Review by tan-man21 — APPROVED** [review]
Looks great!!

**Review by edwardgreenwood38 — COMMENTED** [review]
I only see one migration to add the 'Task' table.  are you still working on the 'List' table?

**Comment by M4RKL4B0Y** [comment]
The lists table is within Tanner's branch. He tackled tables while I handled controllers.

**Review by edwardgreenwood38 — APPROVED** [review]
code looks good. will wait on Tanners merge to get the other items.

**Comment by M4RKL4B0Y** [comment]
Thank you sir!

**Pull request merged**